tests/integration/test_script_runner.py

```diff
@@ -388,6 +388,12 @@
         data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
         backend_dependencies=[BackendDependencies.GCS, BackendDependencies.SPARK],
     ),
+    IntegrationTestFixture(
+        name="how_to_connect_to_data_on_azure_blob_storage_using_pandas",
+        user_flow_script="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_azure_blob_storage_using_pandas.py",
+        data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
+        backend_dependencies=[BackendDependencies.AZURE],
+    ),
```

Comment:
Recently we split up test definitions by backend so we could parallelize / isolate in `tests/integration/test_definitions/`, could you move this to `tests/integration/test_definitions/abs/integration_tests.py`?